packages/vkui/src/components/SelectMimicry/SelectMimicry.tsx

```diff
@@ -53,12 +53,13 @@ export const SelectMimicry = ({
       tabIndex={disabled ? undefined : tabIndex}
       className={classNames(
         styles['Select'],
+        'vkuiInternalSelect',
         sizeY !== SizeType.REGULAR && sizeYClassNames[sizeY],
         !children && styles['Select--empty'],
         multiline && styles['Select--multiline'],
         align &&
```

Comment:
```suggestion
        align && align !== "left"
```
и можно будет удалить `left: null`

Reply:
✅